base_content_space.sol

```diff
@@ -388,16 +451,41 @@ BaseCtFactory20200928110000PO: Replace tx.origin with msg.sender in some cases
 
 */
 
+contract ContentFactoryHelper {
```

Comment:
How does this work, operationally?  Do we need a `build` script to run to update the bytes?

Reply:
I actually don't have a good operational solution for this. Right now there is a test in `elv-master` that fails if there is a mismatch.